clearwater-cassandra/usr/share/clearwater/clearwater-cluster-manager/plugins/cassandra_plugin.py

```diff
@@ -75,6 +75,16 @@ def join_cassandra_cluster(cluster_view,
         doc["listen_address"] = ip
         doc["seed_provider"][0]["parameters"][0]["seeds"] = seeds_list_str
         doc["endpoint_snitch"] = "GossipingPropertyFileSnitch"
+       
+        # Work out the timeout from the target_latency_us value (assuming
+        # 100000 if it isn't set)
+        get_latency_cmd = "target_latency_us=100000; . /etc/clearwater/config; echo -n $target_latency_us"
+        latency = subprocess.check_output(get_latency_cmd,
+                                          shell=True,
+                                          stderr=subprocess.STDOUT)
+
+        timeout = int(latency) * 4 / 5
```

Comment:
I checked the cpp-common code. The timeout used in HTTP connection is (5 * target_latency). So I think we should set the timeout to (5 * target) * (4/5) = target*4.